streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/controller/ThreadingController.java

```diff
@@ -65,15 +66,37 @@
      * The threading controller
      */
     public static ThreadingController getInstanceLowPriority() {
-        return INSTANCE_LOW_PRIORITY;
+        synchronized (ThreadingController.class) {
```

Comment:
OK.  I don't have strong feelings one way or the other.  Slight preference to using it in the method signature since it is more succinct.